internal/pkg/cli/task_run.go

```diff
@@ -647,6 +673,18 @@ func (o *runTaskOpts) Execute() error {
 		return err
 	}
 
+	update := false
+
```

Comment:
nit: the slightly more idiomatic way of writing it is just:
```suggestion
	var shouldUpdate bool
```